resteasy-jaxrs/src/main/java/org/jboss/resteasy/plugins/delegates/CookieHeaderDelegate.java

```diff
@@ -16,10 +18,40 @@ public Cookie fromString(String value) throws IllegalArgumentException
       return CookieParser.parseCookies(value).get(0);
    }
 
-   public String toString(Cookie value)
+   public String toString(Cookie cookie)
    {
-      StringBuffer buf = new StringBuffer();
-      ServerCookie.appendCookieValue(buf, 0, value.getName(), value.getValue(), value.getPath(), value.getDomain(), null, -1, false);

```

Comment:
Ha ha, hadn't realized current version **always** uses "version 0" (aka Netscape spec, or now RFC 6265) cookies (but strangely includes path and domain).

If this passes the TCK (it's been that way for at least 8 years, so either the TCK doesn't run that code at all, or it doesn't really care about the serialization provided it's properly parsed on the server-side, and because Servlet implementations –which I believe the TCK requires– don't really support RFC 2109, the TCK probably doesn't check the path and domain –and version– attributes), then I see **no reason to change to support RFC 2109.** (but possibly change the code to omit the path and domain, as `ServerCookie.appendCookieValue` will unconditionally add them, irrespective of the version)

What was the reason you changed that code?